htdocs/langs/en_US/cashdesk.lang

```diff
@@ -168,3 +168,30 @@ SumupAffiliate=Sumup affiliate
 SumupAppId=Sumup AppId
 NotAvailableForCountryWhenModuleIsOn=Not available for country %s when module %s is enabled
 Printers=Printers
+Reports=Reports
+MonthlyReport=Monthly Report
+AvailableReports=Available Reports
+TotalForYear=Total for year %s
+NoDataFound=No data found
```

Comment:
No need to add this translation entry. You can reuse NoRecordFound that already exist.